equinor/EchoComponents
# Feature: #49136 remove ayelix dependencies

### Description
Moving ayelix dependency, PageMenuButton to components.
Mostly it's used for displaying round icons on PDFs, for example: like zoom in / out, toggle buttons.

Because it's quite in a bad shape, I refactored it a bit.
What was done:
- Renamed it to `RoundIconButton`
- Removed async feature from it. Reason: it's unused.
- Reworked the variants: `simpleButton` change to `toggleButton`. Removed unused variants.
- Removed unused 'title' prop

### Screenshots
After update on echoPediaWeb:
Action buttons on the PDF look the same as before
![image](https://user-images.githubusercontent.com/13872908/134902311-599a09d5-2860-43ec-843a-482430375a1a.png)

### How to test:
Check out the following branch, and use this echoComponents branch as dependency.
All, previous pageMenuButton's should work as before (like PDF action buttons: Zoom in / out, toggle tags etc.
Related PR in echoPediaWeb:
https://github.com/equinor/EchopediaWeb/pull/840
